taxi/views.py

```diff
@@ -87,3 +88,26 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
```

Comment:
The task requires using `UserCreationForm` for updating users, but the current implementation uses `DriverForm`. Please update this to use `UserCreationForm` to meet the task requirements.